api/src/db/DynamoDataClient.ts

```diff
@@ -16,7 +16,7 @@ export const DynamoDataClient = (
     try {
       let nextToken: string | undefined = undefined;
       let migratedCount = 0;
-      const batchSize = 100;
+      const batchSize = 25;
```

Comment:
[dust] How many in a page of results? I figured we could just do a page of results at a time, since it consumes the same amount of read units. If we do batch sizes of 25 and we have 180,000+ users to migrate (which we'll have every time we add a new migration), that will take a long time.